src/qjs.c

```diff
@@ -1025,3 +1025,106 @@ qjs_string_create_chb(JSContext *cx, njs_chb_t *chain)
 
     return val;
 }
+
+
+JSValue
+qjs_string_hex(JSContext *cx, const njs_str_t *src)
+{
+    JSValue    ret;
+    njs_str_t  dst;
+    u_char     buf[QJS_ENCODE_BUF_LEN];
```

Comment:
We do not need to make `QJS_ENCODE_BUF_LEN ` public, it is a detail of implementation. Here and below.